lib/cm_clickpost/payload_helper.rb

```diff
@@ -0,0 +1,113 @@
+module CmClickpost
+    module PayloadHelper
+  
+      class << self
+
+        def pickup_address_payload(pickup_info = {})
+          {
+              "pickup_state": pickup_info.fetch(:pickup_state),
+              "pickup_address": pickup_info.fetch(:pickup_address),
+              "email": pickup_info.fetch(:email),
+              "pickup_time": pickup_info.fetch(:pickup_time_in_iso_format),
+              "pickup_pincode": pickup_info.fetch(:pickup_pincode),
+              "pickup_city": pickup_info.fetch(:pickup_city),
+              "tin": pickup_info.fetch(:tin),
+              "pickup_name": pickup_info.fetch(:pickup_name),
+              "pickup_country": pickup_info.fetch(:pickup_country, "IN"),
+              "pickup_phone": pickup_info.fetch(:pickup_phone)
+          }
+        end
+
+        def drop_address_payload(drop_info = {})
+          {
+              "drop_address": drop_info.fetch(:drop_address),
```

Comment:
@jish0desarkar - you can add the list of attributes in an array and fetch it via map condition?
For certain attributes, fetch will be different and that is okay I guess.  This way if you are adding additional attributes you just need to add them in array only.